source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -0,0 +1,166 @@
+---
+title: IPv6 subnets on ovirt-provider-ovn
+category: feature
+authors: mdbarroso
+feature_name: Support IPv6 subnets on ovirt-provider-ovn
+feature_modules: ovirt-provider-ovn
+feature_status: In Development
+---
+
+# IPv6 subnets on ovirt-provider-ovn
+
+## Summary
+
+This feature provides support for IPv6 addresses on guest virtual machines
+whose networking is provided by OVN.
+
+Address auto-configuration is - despite being desirable - not a must at this
+stage, and its implementation will take place at a later stage.
+
+Related patches can be found on [gerrit topic:provider_ipv6_subnets](https://gerrit.ovirt.org/#/q/topic:provider_ipv6_subnets).
+
+### Owner
+* Feature Owner: [Miguel Duarte Barroso](https://github.com/maiqueb) (mbarroso in #ovirt)
+* Email: <mdbarroso@redhat.com>
+
+### Terminology
+
+* Networking API: [OpenStack RESTful Networking API](https://developer.openstack.org/api-ref/network/v2/)
+* OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that
+brings Software Defined Networking to OVS.
+* oVirt OVN provider: a proxy that implements a subset of the Networking API
+and interacts with OVN.
+* Subnet: a layer 3 concept, created on top of a network. As per openstack's documentation it is a 'block of IP addresses and associated configuration state. Subnets are used to allocate IP addresses when new ports are created on a network.'
+* Network: an isolated Layer 2 networking segment.
+* DHCP: dynamic host configuration protocol. Client server protocol that assigns an IP address - along with other configuration parameters to the client.
+* MTU: maximum transmission unit. The maximum frame size on an L2 network.
+* MAC: medium access control. Unique identifier assigned to a network interface.
+* RA: router advertisements.
+* IPAM: IP Address Management.
+* guests: Virtual Machines running on an hypervisor.
+
+### Benefit to oVirt
+
+IPv6 addresses are currently supported in oVirt, but the L3 capabilities on OVN
+provided networks are severelly limited, since it is missing IPAM and routing
+capabilities.
+This feature addresses those gaps, by providing IPv6 IPAM capabilities to oVirt
+logical networks, provided by OVN.
+With IPv6 IPAM in place, routing amongst different IPv6 subnets will be
+possible, and achieved through an OVN logical router.
+
+## Objectives
+
+Provide guests (virtual machines) attached to OVN networks IPv6 address support.
```

Comment:
Let's change it to:
 Provide IPv6 address support to guests (virtual machines) attached to OVN networks.

Reply:
Done.